wordpress-mobile/WordPress-iOS
# Jetpack App: Disable new post shortcuts

Resolves #16209 

## Description
This PR disables the New Post and New Photo Post shortcuts for the Jetpack app

WordPress | Jetpack
-- | --
<img src="https://user-images.githubusercontent.com/6711616/113266882-b6c42600-9310-11eb-8c69-eb744cfa1e5d.png" width=300> | <img src="https://user-images.githubusercontent.com/6711616/113383230-702d0500-93be-11eb-83f6-18170ed685d8.png" width=300>

## How to test:
1. Delete installed WordPress app
2. Build WordPress and login
2. Long press on WordPress app icon
3. ✅ App shortcuts should be displayed (Notifications, Stats, New Photo Post, New Post)
4. Delete installed Jetpack app
5. Build Jetpack and login
6. Long press on Jetpack app icon
7. ✅ Only the Notifications and Stats shortcuts should be displayed

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test Jetpack and WordPress schemes

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
